isisd/isis_spf.c

```diff
@@ -2173,15 +2351,16 @@ DEFUN(show_isis_topology, show_isis_topology_cmd,
 #ifndef FABRICD
       " [<level-1|level-2>]"
 #endif
-      ,
+      " [algorithm (128-255)]",
```

Comment:
be careful, Flex Algo is not standardized for fabricd

Reply:
done